### DISCUSSION TITLE
Issues with default playback

### DISCUSSION THREAD

**Discussion by MarcSabatella** [musescore/MuseScore#13337] [General]
There have been a number of issues reported about the playback in MuseScore 4.  It's not clear how many of these issues will apply to Muse Sounds or if that uses an entirely separate playback engine, but I wanted to create a place to discuss and track the issues as a whole.

Most of the issues I hear can be demonstrated with one simple piano piece - my old demo score for MuseScore, "Reunion".  I have attached my current 4.0 version of this as "Reunion-40.zip".  Just change the filename to MSCZ and it's ready to play.  This piece was specifically written to show off as many MuseScore features as possible (back in the 1.2 days), so it makes an excellent test case for both engraving and playback.

[Reunion-40.zip](https://github.com/musescore/MuseScore/files/9558674/Reunion-40.zip)

Here is the link to an older version of this score on musescore.com, so you can hear how it is *supposed* to sound.

https://musescore.com/user/2975/scores/39069

The difference is really quite shocking, and not in a good way.  I'll list them in the comments with links to existing issues where appropriate.

**Comment by MarcSabatella** [comment]
Here are the issues I hear:

1. ***This is definitely improved, but I'm not convinced it's completely right***: The tempo just isn't steady.  Instead of the eighth notes taking 50% of the beat, it's something like 45/55 (just a guess) - a sort of reverse swing that permeates the entire piece.  This is reproduce in a trivial example where the top staff of a piano has quarter notes, the bottom eighths.  Seems to only affect two staves of a single instrument, not separate instruments.  See https://github.com/musescore/MuseScore/issues/12870
~~2. In general, the tenuto markings are playing way too loud (see bar 8, and also the last note of bar 10, but oddly, not the rest of the passage in bars 9-10).  See https://github.com/musescore/MuseScore/issues/12678.  However, it seems this may primarily affect MU3 scores.  If I copy that 3/4 passage into a new MU4, or enter it from scratch, only the first tenuto marking plays too loud - and only when I add the slur into the doubled "D".~~
~~3. Accents (sforzato - Shift+V) are ignored during playback both for MU3 scores and new ones created from scratch.  See https://github.com/musescore/MuseScore/issues/11933~~
4. Pedal markings are not working.  They work if you add a single `|___|` marking to a measure, but the standard pedal change `|___/\___|` results in the pedal being disabled and never re-enabled.  See https://github.com/musescore/MuseScore/issues/12869.  This issue has not yet received a priority label.

~~There is also something quite wrong with the dynamics in bars 15 to the end.  The crescendo to forte in 15-16 is ignored, but then there is suddenly increase in volume *after* the forte that concludes the crescendo, and then the diminuendos don't really seem to do anything after that.  This has no specific issue filed yet.  Trying to get it to a simpler case, as I'm not sure what's triggering it.~~

**Comment by MarcSabatella** [comment]
I think the tenuto issue is really about the combination of tenuto and slurs, at least for scores created in MU4.  For scores created in MU3, it seems more pervasive.

~~It turns out arpeggios have this same issue of playing much too loudly under slurs, and that's the main issue with the dynamics in the last few measures here.  See https://github.com/musescore/MuseScore/issues/13340.~~

~~However, that's not the whole story.  There is still an unexplained bump in volume in the middle of bar 16.  It is somehow connected to the slur in the LH as it goes away when I delete that.  Which makes no sense, and I can't reproduce this elsewhere, *except* temporarily when time signature changes are also involved, which makes no sense either.~~

**Comment by MarcSabatella** [comment]
~~Another issue: chords with arpeggios also mess up the tempo, causing bars 17-20 to sound especially bad.  See https://github.com/musescore/MuseScore/issues/13341~~

**Comment by MarcSabatella** [comment]
This one is fixed now 

~~Another issue affecting playback on this score is https://github.com/musescore/MuseScore/issues/12140 - tempo markings ignored after ritardando or other gradual tempo change markings.  Gradual tempo changes are of course a new feature in MU4, so the original versions of this score did not contain them.  Instead, I faked it with ordinary tempo text.  But I'd like to add the real thing now.  Right now, I'm working around the bug by deliberately ending my gradual tempo changes a note early.~~

**Comment by MarcSabatella** [comment]
Current status from my perspective is that everything is reasonable *except* the pedal.  I am still not sure the unsteadiness of the playback is completely resolved, but at this point I don't trust my ears anymore.  I guess we'll see soon enough if others perceive this to be a problem.

**Comment by IU3BRO** [comment]
is it possible to enlarge (thicken) the vertical blue bar running on score during playback, like it was in MU3, and maybe changing note color to RED, like Myriad Harmony Assistant makes (MU3 changes notes in blue during playback). Thanks Marc de Tilio asiviero1@gmail.com
